docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.1'
+
+services:
+
+  rds.acmucsd.local:
+    container_name: pg.keycons.local
```

Comment:
Is there any reason the hostname and container name need to match prod? Can't we just call this `postgres`?